packages/apify/src/actor.ts

```diff
@@ -441,22 +441,26 @@ export class Actor<Data extends Dictionary = Dictionary> {
      *
      * @ignore
      */
-    async reboot(): Promise<void> {
+    async reboot(options: RebootOptions = {}): Promise<void> {
```

Comment:
Pls can we describe the use case of reboot() in jsdoc here? E.g. if the actor gets into irrecoverable error state, it can just reboot itself and continue work where it left off.